app/services/exports/export_distributions_csv_service.rb

```diff
@@ -63,8 +65,11 @@ def base_table
         "Source Inventory" => ->(distribution) {
           distribution.storage_location.name
         },
-        "Total Items" => ->(distribution) {
-          distribution.line_items.total
+        "Total number of Items" => ->(distribution) {
+          # filter the line items by item id (for selected item filter) to
+          # get the number of items
+          distribution.line_items.find_by(item_id: @filtered_item_id)&.quantity ||
```

Comment:
I believe there can be more than one line item _for the same item kind_ within a single distribution, in which case we'd want to use `where` rather than `find_by` (thanks @heyapricot!) and then `:total` instead of `:quantity`.

Reply:
yeah I can change that. However when I edited a distribution and added an existing line item to it, it gets added to the previous one, not creating a separate line item entry to the distribution though.